src/components/specific/groups/group-import/GroupImport.vue

```diff
@@ -16,107 +16,203 @@
       />
     </BIMDataButton>
     <template v-if="isOpen">
-      <BIMDataMenu
-        class="group-import__menu"
-        childrenLeft
-        :menuItems="projectsToDisplay"
-        subListMaxHeight="300px"
-        maxHeight="500px"
-        v-if="projectsToDisplay.length"
-      >
-        <template #item="{ item }">
-          <div
-            class="flex items-center justify-between"
-            :class="{ warning: isWarning(item) }"
-            style="width: 100%"
-          >
-            <BIMDataTextbox width="70%" :text="item.text" :tooltip="false" />
-            <div class="flex items-center">
-              <template v-if="isWarning(item)">
-                <BIMDataIconWarning
-                  class="group-import__menu__warning"
-                  color="high"
-                  size="xxs"
-                  fill
-                />
-              </template>
-              <BIMDataIconChevron size="xxs" margin="0 0 0 6px" />
-            </div>
+      <div class="group-import__menu">
+        <BIMDataSearch
+          v-if="projectsToDisplay.length"
+          width="200px"
+          radius
+          :placeholder="$t('t.search')"
+          v-model="searchText"
+        />
+        <!-- Template if no project in space  -->
+        <template v-if="projectsToDisplay.length === 0">
+          <div class="group-import__menu--empty p-6 color-granite">
+            <span>
+              {{ $t("GroupImport.noProject") }}
+            </span>
           </div>
         </template>
-
-        <template #children="{ item }">
-          <div v-if="isWarning(item)" class="group-import__children__no-child">
-            <div class="group-import__children__no-child__content">
-              <BIMDataIconWarning color="high" size="xs" fill />
-              {{ item.children[0].text }}
-            </div>
+        <!-- Template if no group when searching  -->
+        <template v-if="filteredProjectsToDisplay.length === 0">
```

Comment:
It should be a `v-else-if` here in order to avoid both messages to be displayed simultaneously.